the-grid.html

```diff
@@ -149,6 +149,17 @@
                         value: false,
                         reflectToAttribute: true,
                         observer: '_attachEvents'
+                    },
+
+                    /**
+                     * Enable infinite automatic extension of the grid on resize and move.
+                     * @type {boolean}
+                     */
+                    selfResize: {
```

Comment:
@yorrd we either need to store those counts in internal vars or better determine those col and row counts on the fly so we can compute the stylesheet. I would prefer the second proposal if it does not affect performances. Perhaps we will ne to pass row and col counts as parameter in `#computeStyles`